package/gradle/verification-metadata.xml

```diff
@@ -210,5 +210,13 @@
             <sha256 value="b51f8867c92b6a722499557fc3a1fdea77bdf9ef574722fe90ce436a29559454" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.junit" name="junit-bom" version="5.9.3">
```

Comment:
this should be in `package/verification-template/gradle/verification-metadata.xml` instead of this file